doc/tutorials/klondike/app/lib/step4/klondike_game.dart

```diff
@@ -45,19 +45,15 @@ class KlondikeGame extends FlameGame {
       ),
     );
 
-    final world = World()
-      ..add(stock)
-      ..add(waste)
-      ..addAll(foundations)
-      ..addAll(piles);
-    add(world);
+    world.add(stock);
```

Comment:
I'm curious why the replacement of cascade `add`s with individual `add`s. Also, in the transformation, you lost adding `world` to the current game component.

Reply:
That change is to comply with Flame 1.9.x, where `world` and `camera` objects are pre-built into FlameGame and you don't need to do anything with `world` other than add components to it IIUC. Re cascade syntax, this Tutorial is aimed at programmers who are new to Flame and perhaps also to Dart, so I think more explicit syntax, which is also used in other OO languages is the way to go.

Reply:
Ahhh, this makes sense. Thanks for the explanation!